docs/business.yaml

```diff
@@ -2712,41 +3097,62 @@ components:
       description: 'Add or remove a director, or update information regarding a current director.'
```

Comment:
Can you help to add `title` for this schema (and for change of address) like that is done for AGM Extension (line 2463)?

Reply:
Sure